### PR TITLE
Tweak pre-commit message

### PR DESCRIPTION
**Pull request by zanieb**
Follow-up to https://github.com/astral-sh/ruff/pull/6153

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -69,13 +69,19 @@ and pre-commit to run some validation checks:
...
 when making a commit:
 
 ```shell
 pre-commit install
 ```
 
+or you can run pre-commit manually as needed:
+
+```shell
+pre-commit run --all-files
```
Comment (konstin) [inline CONTRIBUTING.md:82]: that's now duplicated with line 93|99

Comment (zanieb) [reply CONTRIBUTING.md:82]: Oh true. Do you think it's excessive?

Comment (konstin) [reply CONTRIBUTING.md:82]: i'd remove either, i think it's important to keep the part that's required for a successful first PR as short as possible

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -69,13 +69,19 @@ and pre-commit to run some validation checks:
...
+```
+
```
Comment (zanieb) [inline CONTRIBUTING.md:84]:
```suggestion
```

**Review by konstin — APPROVED** [review]

**Pull request merged**